packages/static-hosting/lib/static-hosting.ts

```diff
@@ -63,6 +67,7 @@ export class StaticHosting extends Construct {
             new CfnOutput(this, 'S3LoggingBucketName', {
                 description: "S3 Logs",
                 value: s3LoggingBucket.bucketName,
+                exportName: `${exportPrefix}S3LoggingBucketName` 
```

Comment:
What was the default value of exportName before this change? As in if I upgrade my static hosting stack to this version, will all my resources get re-created with a different name? Ideally we don't want this behaviour.

I'm assuming that's what `const exportPrefix =  props.stackExportPrefix ? props.stackExportPrefix :  'StaticHosting'` is for? If so, maybe add a comment saying _why_ the default value is set to that just to make it clear.

Reply:
There were no default value, it means it is empty. 
The exportPrefix is used, in case if you got two of these stack in a same region, there will be a conflict, if you should search the same export name.

Reply:
Right, that makes sense :slightly_smiling_face: